pkg/api/v1beta3/dynakube/extensions_props.go

```diff
@@ -33,7 +33,7 @@ func (dk *DynaKube) ExtensionsExecutionControllerStatefulsetName() string {
 }
 
 func (dk *DynaKube) ExtensionsCollectorStatefulsetName() string {
-	return dk.Name + "-extensions-collector"
+	return dk.Name + "-opentelemetry-collector"
```

Comment:
is this correct? just kinda conflicts with the name of the func 😅

Reply:
In the DynaKube, there's still the distinction between extensions and telemeteryIngest.

Reply:
Question:
- is `...-opentelemetry-collector` is the name for the otel `Statefulset`?
   - AFAIK `...-extensions-controller` is the extension specific `Statefulset`'s name
- does this change based on if it is for `extensions` or `telemetry-ingest`?
   - IF not, then `...-opentelemetry-collector` shouldn't be in `extensions_props.go` as it is not extensions specific
      - If we still want it in the `dynakube` package (which IMO we shouldn't), then it should be called something like `OtelCollectorStatefulsetName()`

Reply:
we share the name, but decided to not rename it. I agree we should rename it. Just it's breaking change for users who already using Extensions with 1.4.x

Reply:
my comment was about the func name, I don't know about `breaking change for users who already using Extensions with 1.4.x` part 😅

Reply:
my bad, i read it incorrectly. let me fix it.

Reply:
Okey:
- renamed function to `OtelCollectorStatefulsetName`
- moved to `otelc_props.go` same name as package for collector
- keep old string (for backward compatibility will be handle separately, it was renamed by mistake)

commit 8160eb173